css-multicol-1/Overview.bs

```diff
@@ -150,39 +150,50 @@ Introduction</h2>
 <h2 id="the-multi-column-model">
 The multi-column model</h2>
 
-	A <dfn lt="multi-column container|multicol container" oldids="multi-column-element" export>multi-column container</dfn>
-	(or <i>multicol container</i> for short)
-	is an element whose 'column-width' or 'column-count'
-	property is not ''column-width/auto'' and therefore acts as a container for
-	multi-column layout.
+    An element whose column-width or column-count property is not auto establishes a <dfn lt="multi-column container|multicol container" oldids="multi-column-element" export>multi-column container</dfn>
+	(or <i>multicol container</i> for short). 
+	It's a regular block container, 
+	that establishes a new [=block formatting context=].
+
+	By default this creates a single anonymous <i>multi-column row</i> (<i>multicol row</i>). There may be multiple multi-column rows if there's an outer fragmentation context, such as e.g. pagination.
+	
```

Comment:
I really want to keep rows as the thing that authors interact with though, as it will be horrible to teach otherwise. Multicol is already the weird thing, by not being part of display. If we can make rows seem similar to rows in flex and grid (and therefore row-gap spaces them out etc.) that would be a lot better.

As far as the author is concerned the row does stack content vertically, in implementor / spec land the line is the actual container but that's because we are worried about spanners. As far as an author is concerned they will have a row, which contains columns which sometimes get broken by a spanner.

Also, pagination creates essentially a new row in this model.

Reply:
Yes, definitely, authors should see rows as makes sense for them. But within the model description...